web_dynamic/static/styles/3-header.css

```diff
@@ -18,9 +18,9 @@ header .logo {
     border-radius: 50%;
     width: 40px;
     height: 40px;
-    margin-top: 17.5px;
+    margin-top: 15px;
     margin-right: 30px;
 }
-.available {
+#api_status.available {
```

Comment:
was this able to override the background color when tested? I had trouble getting that to work, but I didn't try id.class to specify it